scripts/validate_docstrings.py

```diff
@@ -704,7 +713,26 @@ def validate_one(func_name):
         for wrong_import in ('numpy', 'pandas'):
             if 'import {}'.format(wrong_import) in examples_source_code:
                 errs.append(error('EX04', imported_library=wrong_import))
+    return errs, wrns, examples_errs
+
+
+def validate_one(func_name):
```

Comment:
I think the names of the functions could be better. Not sure what would be the clearest, but if we leave this as `validate_one`, it would make things simpler to call the other `perform_validation`, `validation_conditions` or something like that. Or may be that one should be `validate_one`, and this should be `get_validation_data`, or something similar.

Not sure what's best, but the current names don't tell much about what each one does.

Reply:
I like `get_validation_data` it seems to be the most descriptive.